Assignment2/jorge5799-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,40 @@
+{
+    "@context": {
+        "alumno": "http//www.alumnos.upm.es/Alumnos#",
+        "properties": "http//www.alumnos.upm.es/Properties#",
+        "z170083": "alumno:z170083",
+        "fullName": "properties:hasFullName:",
+        "hometown": "properties:hometown:",
+        "age": "properties:age:",
+        "almaMaster": "properties:almaMaster:",
+        "parentsName": "properties:parentsName:",
+        "subjects": "properties:subjects:",
+        "name": "subjects:name",
+        "teachers": "subjects:teachers",
+        "description": "subjects:description"
```

Comment:
These subject attribute definitions work, but semantically they aren't ok. The best way to define them might be defining a new prefix for the subject concept and then use it to define their attributes.
You don't need to change them, but remember to provide good definitions for the future.